### PR TITLE
Add a mapping paths

### PR DESCRIPTION
**Pull request by meyerbaptiste**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A
| License       | MIT
| Doc PR        | N/A

* Revert/Refactor https://github.com/api-platform/core/pull/869
* Revert https://github.com/api-platform/core/pull/1078
* Allow to configure a mapping paths like `symfony/framework-bundle`

### CODE REVIEW

File: src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -109,16 +108,10 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
 
-                ->arrayNode('loader_paths')
+                ->arrayNode('mapping')
```
Comment (teohhanhui) [inline src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php]: This is a BC break.

Comment (soyuka) [inline src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php]: Wasn't this changed on master (ie not in 2.x)?

Comment (meyerbaptiste) [inline src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php]: Why? This feature is not in a stable release.

Comment (teohhanhui) [inline src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php]: Oh yes! :heart:

Comment (teohhanhui) [inline src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php]: My bad...

**Pull request merged**